app/seeders/user_seeder.rb

```diff
@@ -0,0 +1,26 @@
+class UserSeeder
+  USERS = [
+    {
+      username: "member",
+      email: "member@member.com",
+      password: "password",
+      first_name: "Member",
+      last_name: "User",
+      role: "member"
+    },
+     {
+      username: "admin",
+      email: "admin@admin.com",
+      password: "password",
+      first_name: "Admin",
+      last_name: "User",
+      role: "admin"
+    }
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.